test/consistency.jl

```diff
@@ -0,0 +1,95 @@
+using Test
+using CondaPkg
+using PythonCall
+
+# Load the Python module/package
+numpy = pyimport("numpy")
+mrcfile = pyimport("mrcfile")
+
+@testset "Consistency Test" begin
+    function map_test(map_jl::Array{Float32,3}, map_py::Array{Float32,3})
+        permuted_py = permutedims(map_py, (3, 2, 1))
+        @test map_jl == permuted_py
+        # @test map_jl == map_py
```

Comment:
Perhaps this better documents the issue and gives us a test to mark as non-broken if we change this in the future.
```suggestion
        # dimensions permuted due to row-major storage in Python and col-major in Julia
        # see https://github.com/sethaxen/MRCFile.jl/issues/10
        @test_broken map_jl == map_py
        permuted_py = permutedims(map_py, (3, 2, 1))
        @test map_jl == permuted_py
```